assets/css/src/components-public/_resets.scss

```diff
@@ -11,72 +11,94 @@ html {
   box-sizing: inherit;
 }
 
-body {
-  color: rgb(40, 48, 61);
-  font-size: 16px;
-  font-weight: normal;
-  text-align: left;
-}
+.mailpoet_form {
+  body {
+    color: rgb(40, 48, 61);
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 16px;
+    font-weight: normal;
+    text-align: left;
+  }
 
-html,
-body,
-p,
-ol,
-ul,
-li,
-dl,
-dt,
-dd,
-blockquote,
-figure,
-fieldset,
-form,
-legend,
-textarea,
-pre,
-iframe,
-hr,
-h1,
-h2,
-h3,
-h4,
-h5,
-h6 {
-  -moz-osx-font-smoothing: grayscale;
-  -webkit-font-smoothing: antialiased;
-}
+  html,
+  body,
+  p,
+  ol,
+  ul,
+  li,
+  dl,
+  dt,
+  dd,
+  blockquote,
+  figure,
+  fieldset,
+  form,
+  legend,
+  textarea,
+  pre,
+  iframe,
+  hr,
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
+    color: rgb(40, 48, 61);
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 16px;
+    -moz-osx-font-smoothing: grayscale;
+    -webkit-font-smoothing: antialiased;
+    font-synthesis: none;
+    font-weight: normal;
+    letter-spacing: normal;
+    text-align: left;
```

Comment:
Now the text is aligned left in the editor:

![image](https://user-images.githubusercontent.com/9365088/142385996-c0b1ac5b-02f2-4a0b-84b7-7218dd474ab1.png)

Should these resets be applied to the editor at all?

Reply:
Fixed this issue by scoping the styles to the Form Frontend